packages/schema/parse/src/abi/index.ts

```diff
@@ -43,139 +16,3 @@ export function createAbi(): Abi {
     importedEnvTypes: [],
   };
 }
-
-type ImportedDefinition = ImportedObjectDefinition | ImportedModuleDefinition;
-
-export function combineAbi(abis: Abi[]): Abi {
```

Comment:
Just curious here: Why is this function being removed? I think we might want it back since we are migrating to fetch abi instead of schemas from external wrappers

Reply:
Why do we need it? I believe we were just using that for combining query and mutation

Reply:
After checking the code, you're totally right; sorry for the confusion ser